website/source/docs/platform/k8s/helm.html.md

```diff
@@ -377,6 +377,176 @@ to run the sync program.
             "envoy_dogstatsd_url": "udp://127.0.0.1:9125"
           }
         ```
+* <a name="v-meshgateway" href="#v-meshgateway">`meshGateway` <sup>(beta)</sup></a> - Values that configure Connect [Mesh Gateways](/docs/connect/mesh_gateway.html).
+
+  * <a name="v-meshgateway-enabled" href="#v-meshgateway-enabled">`enabled`</a> (`boolean: false`) - If true, gateways will be created and
+    Consul servers will be configured to support them. If setting this in an already running cluster, you'll need to
+    reload the servers.
+
+  * <a name="v-meshgateway-globalmode" href="#v-meshgateway-globalmode">`globalMode`</a> (`string: "local"`) -
+  [Globally configure](/docs/connect/mesh_gateway.html#enabling-gateways-globally) which mode the gateway should run in.
+  Can be set to either "remote", "local", "none" or "".
+  See [Modes of Operation](/docs/connect/mesh_gateway.html#modes-of-operation) for a description of each mode.
+  If set, `connectInject.centralConfig.enabled` should be set to true so that
+  the global config will actually be used.
+  If set to the empty string, no global default will be set and the gateway mode
+  will need to be set individually for each service.
+
+  * <a name="v-meshgateway-replicas" href="#v-meshgateway-replicas">`replicas`</a> (`int: 2`) -
+  How many gateway pods to run.
+
+  * <a name="v-meshgateway-wanaddress" href="#v-meshgateway-wanaddress">`wanAddress`</a> -
+  The nested values define the WAN address registration that the `consul connect envoy` command will use to configure the mesh gateway.
+  See the [command overview](/docs/commands/connect/envoy.html#wan-address) for more details.
+
+      * <a name="v-meshgateway-wanaddress-port" href="#v-meshgateway-wanaddress-port">`port`</a> (`int: 443`) -
+      The port that gets registered for the WAN address.
+
+      * <a name="v-meshgateway-wanaddress-usenodeip" href="#v-meshgateway-wanaddress-usenodeip">`useNodeIP`</a> (`bool: true`) -
+      If true, each Gateway Pod will advertise its Node IP
+      (as provided by the Kubernetes downward API) as the WAN address.
+      This is useful if node IPs are routable from other DCs.
+      `useNodeName` and `host` must be `false` and `""` respectively.
+
+      * <a name="v-meshgateway-wanaddress-usenodename" href="#v-meshgateway-wanaddress-usenodename">`useNodeName`</a> (`bool: false`) -
+      If true, each Gateway Pod will advertise its Node Name
+      (as provided by the Kubernetes downward API) as the wan address.
```

Comment:
I think WAN should be capitalized everywhere? 🤔

Reply:
Except in the actual settings of course

Reply:
Yes I think you're right, I'll run through and change it.